CatNote.Tests/Services/TaskGenericServiceTests.cs

```diff
@@ -133,7 +133,7 @@ public async Task GetById_GetTaskByIncorrectId_ReturnNull()
 
         var cancellationToken = new CancellationToken();
 
-        _mockGenericRepository.Setup(x => x.GetById(taskId, cancellationToken)).ReturnsAsync((TaskEntity)null);
+        _mockGenericRepository.Setup(x => x.GetById(taskId, cancellationToken)).ReturnsAsync((TaskEntity?)null!);
```

Comment:
Use parameter for null, like there
![image](https://github.com/Finka95/CatNote/assets/73075156/1ccca3b0-7e0a-494f-b233-e7412a2a92d9)